Prashant_B/Assingment/Assingment_1.rb

```diff
@@ -0,0 +1,78 @@
+module CountriesData
+    DATA=[[ "India","8","20","230","30000","36","devloped"],
+        ["Aus","7","10","200","12000","20","devloping"],
+        ["Bhutan", "6","15","130","2000","25","devloped"],
+        ["Eng","5","20","150","12000","22","devloping"]]
+    ARRAY_OF_HASHESH=[]
+    def create_hash
```

Comment:
You can directly assign array of hash to DATA constant, 
no need of extra method and each loop